custom_components/econnect_metronet/devices.py

```diff
@@ -48,9 +47,9 @@ def __init__(self, connection, config=None):
 
         # Load user configuration
         if config is not None:
-            self._sectors_home = parse_areas_config(config.get(CONF_AREAS_ARM_HOME))
```

Comment:
These are not needed anymore, `config.get(CONF_AREAS_ARM_HOME)` returns directly a list (e.g. `[3,4]`). Also, the config can be stored in the database only if valid, so we don't need to validate it anymore (it has been validated already).